classes/02class/exercises/c02-network01/raghuandhpokkalath/README.md

```diff
@@ -0,0 +1,47 @@
+# c02-network01
+
+## Commands Execution Output
+
+- Answers for `10.0.0.1/24`:
+
+  - IP in binary: `00001010.00000000.00000000 .00000001`
+  - subnet in binary: `11111111.11111111.11111111.00000000`                                                                          
+  - subnet in decimals: `255.255.255.0`
+  - private/public: `private`
+  - hosts: `254`
+
+- Answers for `192.168.0.1/16`:
+
+  - IP in binary: `11000000.10101000 .00000000.00000001`
+  - subnet in binary: `11111111.11111111.00000000.00000000` 
+  - subnet in decimals: `255.255.0.0`
+  - private/public: `private`
+  - hosts: `65534`
+
+- Answers for `249.165.166.135/30`:
+
+  - IP in binary: `11111001.10100101.10100110.10000111`
+  - subnet in binary: `11111111.11111111.11111111.11111100` 
+  - subnet in decimals: `255.255.255.252`
+  - private/public: `public`
+  - hosts: `2`
+
+- Answers for `236.68.223.18/32`:
+
+  - IP in binary: `11101100.01000100.11011111.00010010`
+  - subnet in binary: `11111111.11111111.111111111.11111111` 
+  - subnet in decimals: `255.255.255.255`
+  - private/public: `public`
+  - hosts: `1`
```

Comment:
Hi Raghu,

From what I see the hosts on your answers refers to usable hosts, if that's the case I believe the number of hosts on this line is incorrect. Could please you change it? Other than that the rest of the exercise if perfect. Keep it up!

Reply:
@christina-roperto sorry number of hosts is 1 right if it is /32 netmask

Reply:
Hi Raghu, my understanding is /32 still refers to one single IP (as you mentioned in your new answer) but it's not a valid subnet. Thanks for amending it! 😄